src/Playwright/Helpers/Driver.cs

```diff
@@ -37,7 +37,17 @@ internal static string GetExecutablePath()
             DirectoryInfo assemblyDirectory = new(AppContext.BaseDirectory);
             if (!assemblyDirectory.Exists || !File.Exists(Path.Combine(assemblyDirectory.FullName, "Microsoft.Playwright.dll")))
             {
+#if NETFRAMEWORK
                 var assemblyLocation = typeof(Playwright).Assembly.Location;
+#else
+
+                var assemblyLocationFileUrl = typeof(Playwright).Assembly.CodeBase;
+                if (string.IsNullOrEmpty(assemblyLocationFileUrl))
```

Comment:
I'm not sure if I should throw there or instead fallback to  `typeof(Playwright).Assembly.Location;´.

Reply:
Looks fine, since we only do it with .net framework: https://docs.microsoft.com/en-us/dotnet/api/system.reflection.assembly.codebase?view=net-6.0 and its always set there.